Python/stack.py

```diff
@@ -0,0 +1,143 @@
+#implementation of stack using OOP
+#stack is used to store and retrive data
+# the process for stack is FILO i.e.first in last out
+class stack:
+    def __init__(self):
+        #constructor for stack
+        self.mylist=[]
+        self.size=0
+        self.maxsize=5
+
+    #function to display elements of stack
+    def display(self):
+        print(self.mylist)
+
+    #function to display length of stack
+    def length(self):
+        print(self.size)
+    #function to add element to stack
+    def push(self,data):
+        if (self.size==self.maxsize):
+            print('stack is full ,stack overflow')
+            return None
+        self.mylist.append(data)
+        self.size+=1
+        print("Data added to stack.")
+    #function to remove elements from stack
+    def pop(self):
+        if(self.size==0):
+            print("Stack is empty..no data can be popped ")
+        else:
+            self.mylist.pop()
+            self.size-=1
+            print("Data removed from stack.")
+
+    #function to display element on top of stack
+    def peek(self):
+        if(self.size==0):
+            print("Stack is empty...therefore no peek")
+        else:
+            print("Topmost element in the stack is: ",self.mylist[self.size-1])
+    #function to check if stack is empty
+    def isEmpty(self):
+        if(self.size==0):
+            print("Stack is empty")
+        else:
+            print("stack is not empty")
+
+s1=stack()
+
+print("Enter option that you want to perform")
+op=int(input("1:push 2:pop 3:display 4:peek 5:length 6:isEmpty : "))
+
+while(op<=6):
+    if(op==1):
+        data=int(input("Enter data to add: "))
+        s1.push(data)
+        
+    elif(op==2):
+        s1.pop()
+        
+    elif(op==3):
+        s1.display()
+        
+    elif(op==4):
+        s1.peek()
+        
+    elif(op==5):
+        s1.length()
+        
+    elif(op==6):
+        s1.isEmpty()
+        
+    else:
+        print("Invalid option.")
+
+    print("Enter option that you want to perform")
+    op=int(input("1:push 2:pop 3:display 4:peek 5:length 6:isEmpty : "))
+    print()
+
+
```

Comment:
PUT ALL THE LINES STARTING HERE IN A SINGLE MULTILINE COMMENT.
Add time and space compexity.